index.js

```diff
@@ -36,6 +36,15 @@ app.post('/login', (req, res) => {
     login(req, res);
 });
 
+app.get('/login', (req, res) => {
```

Comment:
Is it an API endpoint for users to log in?

Reply:
I've changed it to 
res.status(200).json(login(req, res));
And it worked in Postman. I've did something wrong?